Bulls_Cows.py

```diff
@@ -0,0 +1,78 @@
+import random
+
+def generate_list():
+    lst = [i for i in range(10)]
+    lst = random.sample(lst, 4)
```

Comment:
:+1: Nice solution. But change the name of `lst` and maybe even the function. What about `generate_secret` and `secret`?